Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -118,6 +118,8 @@ def cc_radar(cls, args, repository, issue):
                 sys.stdout.write('Existing radar to CC (leave empty to create new radar)')
                 sys.stdout.flush()
                 input = Terminal.input(': ')
+                if input.startswith("https://rdar.apple.com/"):
+                    input = input.removeprefix("https://rdar.apple.com/")
```

Comment:
I think the correct place for this change is actually `Tools/Scripts/libraries/webkitbugspy/webkitbugspy/radar.py`, and just involves adding a new regex to `Tracker.RES`. I would expect that making the change there makes `test_manual_radar_cc_http` still pass